Source/Diagnostics/FlushFormats/FlushFormatPlotfile.cpp

```diff
@@ -339,7 +339,7 @@ FlushFormatPlotfile::WriteWarpXHeader(
 void
 FlushFormatPlotfile::WriteParticles(const std::string& dir,
                                     const amrex::Vector<ParticleDiag>& particle_diags,
-                                    bool isBTD) const
+                                    const amrex::Real time, bool isBTD) const
```

Comment:
Shouldn't line 102 of this file be modified as well? Maybe there are other instances of this function being called?

Reply:
Good news is that the automated tests appear to have caught this :)

Reply:
you are totally right! I should have corrected the bug!